src/app/clients/form/formSections/NumberAdultsCard.tsx

```diff
@@ -32,35 +28,38 @@ const getQuantity = (input: string): number => {
     return -1;
 };
 
-const getNumberAdults = (
+const setNumberAdults = (
     fieldSetter: FieldSetter,
     errorSetter: ErrorSetter,
-    adults: Person[],
-    gender: Gender
+    adults: NumberAdultsByGender,
+    fieldKey: string
```

Comment:
Would it be accurate to say this?
```suggestion
    fieldKey: keyof NumberAdultsByGender
```

Reply:
![image](https://github.com/NorwoodAndBrixtonFoodbank/nbf-website/assets/138508884/b59d9541-bfc2-4c1d-9184-5265f2b9abd9)